### PR TITLE
fix(lint): add 'alt' attribute to img tags

### PR DESCRIPTION
**Pull request by TheAlmightyCrumb**
Added 'alt' attributes to all image tags appearing in the eslint warnings:
image

### CODE REVIEW

File: src/components/landing-page/landing-page.tsx [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const LandingPage: React.FC<LandingPageProps> = ({
             </div>
             <div className={styles.content}>
                 <div className={styles.cols}>
-                    <img className={styles.logo} src={logoURL} />
+                    <img className={styles.logo} src={logoURL} alt="" />
```
Comment (PeterShershov) [inline src/components/landing-page/landing-page.tsx:47]: alt="landing page image"

File: src/components/gift-card/gift-card.tsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export const GiftCard: React.FC<GiftCardProps> = ({ photoURL = defaultPhotoURL,
             className={`${styles.card}${full ? ` ${styles.full}` : ''}`}
             elevation={Elevation.FOUR}
         >
-            <img src={photoURL} />
+            <img src={photoURL} alt="" />
```
Comment (PeterShershov) [inline src/components/gift-card/gift-card.tsx:32]: alt="gift card image"

File: src/components/form-card/form-card.tsx [hunk 3]
```diff
@@ -27,7 +27,7 @@ export const FormCard: React.FC<FormCardProps> = ({
     return (
         <Card className={`${styles.card} ${styles.wrapper}`} elevation={Elevation.FOUR}>
             <div className={`${styles.card} ${styles.header}`}>
-                <img className={styles.logo} src={logoURL} />
+                <img className={styles.logo} src={logoURL} alt="" />
```
Comment (PeterShershov) [inline src/components/form-card/form-card.tsx:32]: alt="form card image"

**Comment by TheAlmightyCrumb** [comment]
@PeterShershov Initially the commit contained texts: https://github.com/codux-demos/blueprint-webapp/pull/12/commits/3c3cde2fd090680f477533e8e65d4b148762564b but after review from @cariowix, it has changed to "empty string for decorative images" as mentioned in the warning itself.

**Review by PeterShershov — APPROVED** [review]
TIL about empty alt attributes.
https://html.spec.whatwg.org/multipage/images.html#a-graphical-representation-of-some-of-the-surrounding-text

**Pull request merged**